### PR TITLE
Remove remaining uses of progress_bar

### LINKED ISSUE #4: Eliminate console_progressbar dependency
The aspire package came with the `console_progressbar` dependency that had been added to the (joint) `environment.yml`. I suspect `tqdm` (that we have already been using) should suffice for all cases. We need to find cases of `console_progressbar` usage and convert to `tqdm`, and remove this dependency eventually.

### PR DESCRIPTION
**Pull request by garrettwrong**
Today's cleanup contribution, Closes #4 .

Most places were unused imports; they will not be missed.

There were a few places `progress_bar` was actually used, which were substituted up to pretty print formatting to `tqdm` (which is already pretty enough for anything practical).

One of the replacements has an example of using a verbosity variable in conjunction with `disable` argument to make the progress bar optional.

The only difference of note between the two packages is that `tqdm` is (correctly) captured by pytest, and so if you want to display the bar when running tests you must run `pytest -s` to enable stdout display, as it was designed.

### CODE REVIEW

File: src/aspire/aspire/utils/compare_stacks.py [hunk 1]
```diff
@@ -62,9 +59,6 @@ def compare_stacks(stack1, stack2, verbose=None, max_error=None):
         if verbose == 0:
             continue
 
-        elif verbose == 1:
```
Comment (janden) [inline src/aspire/aspire/utils/compare_stacks.py:65]: While we're here, can you remove the blank line above so that the if and elif blocks stick together?

**Review by janden — CHANGES_REQUESTED** [review]

File: src/aspire/aspire/utils/compare_stacks.py (hunk 1 above)
Comment (garrettwrong) [reply src/aspire/aspire/utils/compare_stacks.py:65]: sure, I rm'd the blanks around both elif blocks.

**Comment by garrettwrong** [comment]
@junchaoxia , I think you looked at this one last week. Do you think it is ready to go?

**Review by janden — APPROVED** [review]

**Review by junchaoxia — APPROVED** [review]
I think it is ready to merge.

**Comment by garrettwrong** [comment]
Great thanks!

**Pull request merged**